packages/ra-ui-materialui/src/layout/Notification.tsx

```diff
@@ -78,6 +79,11 @@ export const Notification = (props: NotificationProps) => {
         setOpen(false);
     }, [setOpen]);
 
+    const closeNotificationContext = React.useMemo(
```

Comment:
Why not use `useCallback` here?

Reply:
I don't even need this line actually, `handleRequestClose` is already wrapped in a `useCallback`